.travis.yml

```diff
@@ -98,6 +98,7 @@ jobs:
         tags: true
         repo: iterative/dvc
   - name: Snapcraft snap
+    stage: deploy
```

Comment:
That initial "default stage" change came to bite us in the bum, eh? :slightly_smiling_face: 

Btw, why doesn't previous `stage: deploy` work?

Reply:
updated the PR description to be more explicit why. Think it's because the previous job didn't run.